src/promptflow-evals/pyproject.toml

```diff
@@ -0,0 +1,114 @@
+[build-system]
+requires = ["poetry-core>=1.5.0"]
+build-backend = "poetry.core.masonry.api"
+
+# poetry
+[tool.poetry]
+name = "promptflow-evals"
+version = "0.1.0b1"
+description = "Prompt flow evals"
+license = "MIT"
+authors = [
+  "Microsoft Corporation <aml-pt-eng@microsoft.com>"
+]
+repository = "https://github.com/microsoft/promptflow"
+homepage = "https://microsoft.github.io/promptflow/"
+readme = ["README.md"]
+keywords = ["telemetry"]
+classifiers = [
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+]
+packages = [
+  { include = "promptflow" }
+]
+
+[tool.poetry.urls]
+"Bug Reports" = "https://github.com/microsoft/promptflow/issues"
+
+# dependencies
+[tool.poetry.dependencies]
+python = "<4.0,>=3.8"
+azure-ai-ml = ">=1.14.0"
+promptflow = "<2.0.0"
+promptflow-tools = "<2.0.0"
+
+[tool.poetry.group.dev.dependencies]
+pre-commit = "*"
+import-linter = "*"
+
+[tool.poetry.group.test.dependencies]
+pytest = "*"
+pytest-asyncio = "*"
+pytest-cov = "*"
+pytest-mock = "*"
+pytest-xdist = "*"
+
+# test: pytest and coverage
+[tool.pytest.ini_options]
+markers = [
+  "unittest",
+  "e2etest",
+]
+# junit - analyse and publish test results (https://github.com/EnricoMi/publish-unit-test-result-action)
+# durations - list the slowest test durations
+addopts = """
+--junit-xml=test-results.xml \
+--dist loadfile \
+--log-level=info \
+--log-format="%(asctime)s %(levelname)s %(message)s" \
+--log-date-format="[%Y-%m-%d %H:%M:%S]" \
+--durations=5 \
+-ra \
+-vv
+"""
+env = [
+]
+testpaths = ["tests"]
+
+[tool.coverage.run]
+concurrency = ["multiprocessing"]
+source = ["promptflow"]
+omit = [
+  "__init__.py",
+]
+
+[tool.black]
+line-length = 120
+
+# import linter
+# reference: https://pypi.org/project/import-linter/
+[tool.importlinter]
+root_package = "promptflow"
+include_external_packages = "True"
+
+[[tool.importlinter.contracts]]
+name = "Contract forbidden modules"
+type = "forbidden"
+source_modules = ["promptflow.evals"]
+forbidden_modules = [
```

Comment:
@crazygao  please help to review this part

Reply:
Does eval not use anything inside core or executor or sdk?

Reply:
By the way, change the linter if eval wants to enable these checks.

Reply:
> By the way, change the linter if eval wants to enable these checks.
@crazygao: We'd like to enable the import checks. Could you pleaes elaborate on how to enable it?

Reply:
> Does eval not use anything inside core or executor or sdk?
Yes, eval will have a dependency on core, connection and protentially a few others as well. I've updated the list based on my best knowledge. Will revisit it later on.

Reply:
> > By the way, change the linter if eval wants to enable these checks.
> > @crazygao: We'd like to enable the import checks. Could you pleaes elaborate on how to enable it?

Figured out. https://github.com/microsoft/promptflow/blob/main/.github/workflows/promptflow-import-linter.yml